src/_posts/databases/postgresql/2000-01-01-upgrading.md

```diff
@@ -0,0 +1,233 @@
+advise to take extra care when dealing with them:
+
+- First, carefully read the changelogs provided by PostgreSQL® (we usually link
+  them in our respective changelog entries). Identifying noticeable changes,
+  especially ones that may have a negative impact on your database or app, will
+  allow you to update your application accordingly.
+
+- We also generally advise to test the changes in a [Review App]({% post_url platform/app/2000-01-01-review-apps %}).
+
+  The use of Review Apps can be deactivated on your production application, [as
+  we recommend]({% post_url platform/app/2000-01-01-review-apps %}#configuration-of-review-apps)
+  in our documentation. In general, it is recommended that you carry out this
+  process on a staging application using the same code repository, but linked
+  to a database with no production or customer data. Once you have activated
+  the Review Apps feature, you can create a pull request in which you can
+  modify the `scalingo.json` manifest file to force the deployment of a
+  specific version of PostgreSQL®. The full process is as follow:
+
+  1. Create a new app dedicated to this major-upgrade.
+  2. Link this app to your application's code repository.
+  3. Make sure to enable Review Apps for this application.
+  4. Leverage the [`scalingo.json` manifest file]({% post_url platform/app/2000-01-01-app-manifest %})
+     to:
+     - Specify the version of the database addon you require. This version
+       should match the one you are using in your production environment.
+     - Ideally ask the platform to fill your database with **testing data**,
+       using a [`first-deploy` script]({% post_url platform/app/2000-01-01-app-manifest %}#deployment-hooks).
+
+     Here is an example of a manifest file asking the platform to provision a
+     PostgreSQL `14.6.0` addon and to run the `db-seed.sh` script after the
+     first deployment (the script must be included in your codebase):
+     ```json
+     {
+       "addons": [
+         {
+           "plan": "postgresql:postgresql-starter-512",
+           "options": {
+             "version": "14.6.0"
+           }
+         }
+       ],
+
+       "env": {
+         "CANONICAL_HOST_URL": {
+           "generator": "url"
+         }
+       },
+
+       "scripts": {
+         "first-deploy": "bash -c db-seed.sh"
+       }
+     }
+     ```
+  5. Create a new Pull Request from the repository hosting your application's
+     code. A new Review App is created with the appropriate database version.
+     If you have a `first-deploy` script in your `scalingo.json` manifest, it's
+     executed.
+  6. (optional): If you couldn't use a `first-deploy` script, it's now
+     time to fill your database with your test dataset.
+  7. Once filled with testing data, upgrade your database until it reaches the
+     targeted version.
+  8. Execute your tests scenarii on the Review App. This should help you
```

Comment:
From my dictionary, "scenarii" is a French word (https://www.wordreference.com/fren/scenarii). Even in French its usage is outdated (https://fr.wiktionary.org/wiki/scenarii) ^^
```suggestion
8. Execute your tests scenarios on the Review App. This should help you
```

Reply:
Applied in #2885